internal/bminventory/inventory_test.go

```diff
@@ -5374,6 +5377,10 @@ var _ = Describe("Register AddHostsCluster test", func() {
 		mockMetric.EXPECT().ClusterRegistered(common.TestDefaultConfig.OpenShiftVersion, clusterID, "Unknown").Times(1)
 		mockVersions.EXPECT().GetSupportedVersionFormat(common.TestDefaultConfig.OpenShiftVersion).Return(common.TestDefaultConfig.OpenShiftVersion, nil).Times(1)
 		res := bm.RegisterAddHostsCluster(ctx, params)
+		actual := res.(*installer.RegisterAddHostsClusterCreated)
+
+		Expect(actual.Payload.HostNetworks).To(Equal(defaultHostNetworks))
```

Comment:
may just not equal nil? 
```suggestion
		Expect(actual.Payload.HostNetworks).NotTo(BeNil())
		Expect(actual.Payload.Hosts).NotTo(BeNil())
```

Reply:
Empty slice is both not nil and default value for HostNetworks. In order to avoid the case of different default value I set it to an empty slice.